app/assets/stylesheets/components/_mfa-added-list.scss

```diff
@@ -0,0 +1,5 @@
+.usa-icon-list__item + lg-webauthn-input,
+lg-webauthn-input + .usa-icon-list__item {
+  padding-top: 0.75rem;
+  display: flex;
+}
```

Comment:
What do these styles do? Are these for spacing between icon list items? I'd wonder if we need it at all, since we should be able to rely on default spacing from the list component. Worst case we could use some [design system spacing utility classes](https://designsystem.digital.gov/utilities/margin-and-padding/).

Reply:
I'm not entirely clear why but some MFA are rendered inside of an <li> but others are nested inside of a <lg-webauthn-input> tag. 
![Screenshot 2023-08-01 at 2 30 12 PM (2)](https://github.com/18F/identity-idp/assets/135744319/9b1b5c26-5a11-44b4-abae-762c3b080880)
I don't mind it without the spacing but it's spaced wider in the Figma design.
![Screenshot 2023-08-01 at 2 31 59 PM (2)](https://github.com/18F/identity-idp/assets/135744319/4d79b33d-9b12-4bc2-95bd-24e4e725e43e)